app/js/lib/common/errors.js

```diff
@@ -0,0 +1,8 @@
+export function HttpError(statusCode, message) {
+  this.message = message;
```

Comment:
If this is always going to be about HTTP errors, can you set the message here so that we only need a status code to instantiate the error?
```js
throw new HttpError(404);
throw new HttpError(res.status);
```